app/helpers/alchemy/elements_helper.rb

```diff
@@ -125,7 +125,7 @@ def render_elements(options = {})
     #
     # == Usage
     #
-    #   <%= render_element(Alchemy::Element.available.named(:headline).first) %>
+    #   <%= render_element(Alchemy::Element.named(:headline).select(&:public?).first) %>
     #
```

Comment:
I think we should use the `published` scope here. The `select` only makes sense if we properly eager load. Alternatively we could advocate the ElementsRepository